src/pages/index.js

```diff
@@ -2,37 +2,95 @@ import FormValidator from "../components/FormValidator.js";
 import Card from "../components/Card.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupConfirmDeletion from "../components/PopupConfirmDeletion.js";
 import Section from "../components/Section.js";
-import { initialCards } from "../utils/utils.js";
 import { formValidationConfig } from "../utils/utils.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 import "./index.css";
 
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const addCardModal = document.querySelector("#add-card-modal");
+const changeAvitarModal = document.querySelector("#edit-avatar-modal");
+
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
 const profileEditForm = profileEditModal.querySelector(".modal__form");
 const addCardForm = addCardModal.querySelector(".modal__form");
 const addNewCardButton = document.querySelector(".profile__add-button");
+const editAvatarButton = document.querySelector(".profile__avatar-button");
+const avatarFormElement = changeAvitarModal.querySelector(".modal__form");
 
 function handleImageClick(name, link) {
   previewImagePopup.open(name, link);
 }
 
-function handleProfileEditSubmit(data) {
-  userInfo.setUserInfo(data);
-  editProfilePopup.close();
+function handleProfileEditSubmit(userData) {
+  editProfilePopup.renderLoadingText(true);
+  api
+    .updateUserInfo(userData)
+    .then((res) => {
+      userInfo.setUserInfo({
+        title: res.name,
+        description: res.about,
+      });
+      editProfilePopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      editProfilePopup.renderLoadingText(false);
+    });
+}
+
+function handleAvatarSubmit(link) {
+  editAvatarPopup.renderLoadingText(true);
+  api
+    .updateAvatar(link)
+    .then((res) => {
+      userInfo.updateAvatar(res.avatar);
+      editAvatarPopup.close();
+      avatarFormElement.reset();
+      avatarFormValidator._disableButton();
```

Comment:
`_disableButton`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.